examples/peripherals/uart_async_rxtxtasks/main/uart_async_rxtxtasks_main.c

```diff
@@ -0,0 +1,75 @@
+/* UART asynchronous example, that uses separate RX and TX tasks
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_system.h"
+#include "esp_log.h"
+#include "driver/uart.h"
+#include "soc/uart_struct.h"
+#include "string.h"
+
+static const int RX_BUF_SIZE = 1024;
+
+#define TXD_PIN (4)
+#define RXD_PIN (5)
```

Comment:
Even if this is like that in `uart_echo_example_main.c` example, how about using `GPIO_NUM_4` and `GPIO_NUM_5` instead of bare numbers 4 and 5?
This tells the user directly that we are entering GPIO numbers here.

Reply:
Thanks. This should now be fixed in 0162b3b.